packages/rocketchat-importer/server/startup/setImportsToInvalid.js

```diff
@@ -1,9 +1,16 @@
-/* globals Importer */
+import { Imports } from '../models/Imports';
+import { RawImports } from '../models/RawImports';
+
 Meteor.startup(function() {
 	// Make sure all imports are marked as invalid, data clean up since you can't
 	// restart an import at the moment.
-	Importer.Imports.update({ valid: { $ne: false } }, { $set: { valid: false } }, { multi: true });
+	Imports.update({ valid: { $ne: false } }, { $set: { valid: false } }, { multi: true });
```

Comment:
Why update if going to just drop?

Reply:
`Imports` is the information regarding the progress of the import which we need to have around still so we can see where and why an import failed. `RawImports` is what is being dropped, as it contains all the "raw" data an import has in it such as every single little message.

Reply:
Ah!  Got it!  :+1: